packages/core/src/external/fhir/patient/index.ts

```diff
@@ -13,9 +13,10 @@ import { getIdFromSubjectId, getIdFromSubjectRef } from "../shared";
 
 export type PatientIdAndData = Pick<Patient, "id" | "data">;
 
-const genderMapping: { [k in GenderAtBirth]: "female" | "male" } = {
+const genderMapping: { [k in GenderAtBirth]: "female" | "male" | "unknown" } = {
```

Comment:
Can we refer to the `GenderCodes` type so we don't need to specify them individually?

nit: easier to use `Record<GenderAtBirth, GenderCodes>` than { [k in GenderAtBirth...`